src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
```

Comment:
I'm curious about the move to specifying a `className` instead of an appearance. Since this component defines the class names I think it makes sense to instead ask it to use the right class rather than have each caller need to know the class to add.

If you don't like the `appearance` prop, you could have boolean props for the classes too.

```jsx
<Button alert>Cancel</Button>
<Button action>Continue</Button>